internal/clients/init.go

```diff
@@ -9,20 +10,18 @@ package clients
 import (
 	"fmt"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/coredata"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logging"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/metadata"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/types"
 	"net"
 	"net/http"
-	"strconv"
 	"sync"
 	"time"
 
 	"github.com/edgexfoundry/device-sdk-go/internal/common"
 	"github.com/edgexfoundry/device-sdk-go/internal/config"
-	"github.com/edgexfoundry/device-sdk-go/internal/registry"
-	"github.com/edgexfoundry/go-mod-core-contracts/clients/coredata"
-	"github.com/edgexfoundry/go-mod-core-contracts/clients/logging"
-	"github.com/edgexfoundry/go-mod-core-contracts/clients/metadata"
-	"github.com/edgexfoundry/go-mod-core-contracts/clients/types"
-	consulapi "github.com/hashicorp/consul/api"
+	"github.com/edgexfoundry/device-sdk-go/internal/endpoint"
```

Comment:
it's a minor comment
we was asking to separate the import from standard libraries and GitHub libraries alphabetically from community.
please see the example
https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/clients/logging/logger.go

Reply:
OK, but when I run "go fmt" it sorts them alphabetically with the standard libraries.